src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -117,7 +117,13 @@ def __call__(self, parser, namespace, values, option_string=None):
                             existing_values = getattr(namespace, arg.name, None)
                             if existing_values is None:
                                 existing_values = IterateValue()
-                            existing_values.append(parts[arg.id_part])
+                                existing_values.append(parts[arg.id_part])
+                            elif isinstance(existing_values, str):
+                                logger.warning(
+                                    "Property '%s=%s' being overriden by value '%s' from IDs parameter.", # pylint: disable=line-too-long
+                                    arg.name, existing_values, parts[arg.id_part]
+                                )
+                                existing_values = [existing_values] # pylint: disable=redefined-variable-type
```

Comment:
In this 'else' case, where do you assign `parts[arg.id_part]` to `existing_values`?